api/tasks.py

```diff
@@ -10,9 +11,11 @@ def create_delivery(order):
     deliveries_by_vendor = Client(order).create_delivery()
     for delivery in deliveries_by_vendor:
         if not delivery._is_errored():
-            delivery.send_mail_to_vendor()
+            email = SuccessDeliveryMail(delivery)
+            email.send()
         else:
-            delivery.send_error_mail_to_admin()
+            email = FailedDeliveryMail(delivery)
+            email.send()
```

Comment:
I think the local var is superfluous here. We could perfectly do `FailedDeliveryMail(delivery).send()`, but it's just a minor thing.